packages/desktop-client/src/components/payees/PayeeTableRow.tsx

```diff
@@ -45,12 +46,10 @@ function RuleButton({ ruleCount, focused, onEdit, onClick }: RuleButtonProps) {
         onSelect={onClick}
       >
         <Text style={{ paddingRight: 5 }}>
-          {ruleCount > 0 ? (
-            <>
-              {ruleCount} associated {ruleCount === 1 ? 'rule' : 'rules'}
-            </>
+          {count > 0 ? (
+            <Trans count={count}>{{ count }} associated rules</Trans>
```

Comment:
Please could we leave the variable names as-is and alter the <Trans> component instead?

```suggestion
            <Trans count={ruleCount}>{{ count }} associated rules</Trans>
```

Reply:
Done!